src/test/java/eu/dissco/backend/repository/AnnotationRepositoryIT.java

```diff
@@ -0,0 +1,299 @@
+package eu.dissco.backend.repository;
+
+import static eu.dissco.backend.TestUtils.USER_ID_TOKEN;
+import static eu.dissco.backend.TestUtils.givenAnnotationJsonApiData;
+import static eu.dissco.backend.TestUtils.givenAnnotationJsonApiDataList;
+import static eu.dissco.backend.TestUtils.givenAnnotationResponse;
+import static eu.dissco.backend.database.jooq.tables.NewAnnotation.NEW_ANNOTATION;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.backend.domain.AnnotationResponse;
+import eu.dissco.backend.domain.JsonApiData;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+import org.jooq.JSONB;
+import org.jooq.Query;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AnnotationRepositoryIT extends BaseRepositoryIT {
+
+  private AnnotationRepository repository;
+
+  @BeforeEach
+  void setup() {
+    ObjectMapper mapper = new ObjectMapper().findAndRegisterModules();
```

Comment:
Could use the TestUtils mapper, and add the SerializationInclusion there I think